scripts/core/config.ts

```diff
@@ -3,8 +3,12 @@ import {appConfig} from 'appConfig';
 import {IArticle} from 'superdesk-api';
 
 export function isMediaEditable(item?: IArticle) {
-    if (item !== undefined && item._fetchable !== undefined) {
-        return item._fetchable;
+    if (item !== undefined) {
```

Comment:
I'd use `!= null`. It checks for `null` and `undefined` at the same time(because of single equals, as opposed to `!==`).